likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/post/style/LMFeedPostViewStyle.kt

```diff
@@ -20,26 +22,35 @@ import com.likeminds.feed.android.core.utils.model.LMFeedPadding
  * [LMFeedPostViewStyle] helps you to customize the header view
  *
  * @property postHeaderViewStyle : [LMFeedPostHeaderViewStyle] this will help you to customize the post header view [LMFeedPostHeaderView]
- * @property postContentTextStyle : [LMFeedPostContentViewStyle] this will help you to customize the text content of the post and the keyword searched in post
+ * @property postContentTextStyle : [LMFeedPostContentViewStyle] this will help you to customize the text content of the post and the keyword searched in post text
+ * @property postHeadingContentStyle : [LMFeedPostContentViewStyle] this will help you to customize the heading of the post and the keyword searched in post heading
  * @property postMediaViewStyle: [LMFeedPostMediaViewStyle] this will help you to customize the media attached in the post
  * @property postActionViewStyle: [LMFeedPostActionViewStyle] this will help you to customize the action view of the post [LMFeedPostActionView]
  * @property postTopicsGroupStyle: [LMFeedChipGroupStyle] this will help you to customize the chip group in the post [LMFeedChipGroup]
  * @property postTopicChipsStyle: [LMFeedChipStyle] this will help you to customize the chips of the chip group in the post [LMFeedChip]
+ * @property postTopResponseViewStyle: [LMFeedPostTopResponseViewStyle] this will help you to customize the top response in the post [LMFeedPostTopResponseView]
+ * @property postTopResponseViewStyle: [LMFeedPostTopResponseViewStyle] this will help you to customize the prompt to answer view [LMFeedLabelImageContainerView]
  *
  * */
 class LMFeedPostViewStyle private constructor(
-    //post header style
+    // post header style
     val postHeaderViewStyle: LMFeedPostHeaderViewStyle,
     // post text content style
     val postContentTextStyle: LMFeedPostContentViewStyle,
-    //post media style
+    // post heading style
+    val postHeadingContentStyle: LMFeedPostContentViewStyle?,
+    // post media style
```

Comment:
Why we need separate key for heading all the heading related views can go inside `LMFeedPostContentViewStyle`